types/sfcc-ci/index.d.ts

```diff
@@ -0,0 +1,64 @@
+// Type definitions for sfcc-ci 2.9
+// Project: https://github.com/SalesforceCommerceCloud/sfcc-ci (Does not have to be to GitHub, but prefer linking to a source code repository rather than to a project website.)
+// Definitions by: Karim Shehadeh <https://github.com/kshehadeh>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export type SfccCiCallback = (err: any, result: any) => any;
+
+export const auth: {
+    auth: (client_id: string, client_secret: string, callback: SfccCiCallback) => void;
+};
+export const cartridge: {
+    add: (instance: any, cartridgename: any, position: any, target: any, siteid: any, verbose: any, token: any, callback: any) => void;
```

Comment:
```suggestion
    add: (instance: any, cartridgename: any, position: any, target: any, siteid: any, verbose: any, token: any, callback: SfccCiCallback) => void;
```
is this the same callback signature?